lighthouse-core/lib/lh-error.js

```diff
@@ -169,6 +175,12 @@ const ERRORS = {
     message: strings.requestContentTimeout,
   },
 
+  // Protocol timeout failures
+  SECURITY_STATE_TIMEOUT: {
```

Comment:
in the usual case for the security state check, it should just reject on an insecure security state, right? If so, we probably want to make this a more general protocol communication timeout error (anticipating #6296)

Reply:
My plan was to modify that bit in the referenced issue. but now that we have this concrete proto definition it makes sense to make it good sooner rather than later

Reply:
Oh, I actually removed the timeout stuff for security checking. I'll remove this too.